### DISCUSSION TITLE
WHY? a lot of garbled code in the result document!

### DISCUSSION THREAD

**Discussion by theone1978** [magicpdf/Magic-PDF#821] [Q&A]
What is the problem with a lot of garbled code in the result document, do you need a special output method, and how should you solve it?
![1730339207355](https://github.com/user-attachments/assets/d0e12dad-e665-4af8-8e74-2102bb3f7793)

**Comment by myhloli** [comment]
You should enable the formula rendering function in your Markdown viewer to ensure that LaTeX formulas are rendered correctly.
